security/network_defense.adoc

```diff
@@ -0,0 +1,78 @@
+[[security-network]]
+= Network Defense
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-network-isolating-applications]]
+== Isolating Applications from Each Other
+
+Deploying modern microservices-based applications in containers often means
+deploying multiple containers distributed across multiple nodes. You need a way
+to isolate applications from other applications within a cluster. You need to
+consider network defense.
+
+A typical public cloud container service, such as Google Container Engine (GKE),
+Azure Container Services, and AWS Container Service, are single tenant services.
+They let you run your containers on the virtual machine (VM) cluster that you
+spin up. {product-title} allows you to take a single Kubernetes cluster and
+segment the traffic to make multi-tenant clusters the isolate different users,
+teams, applications and environments within a single cluster.
```

Comment:
How does this help security? What your describing here or with this section seem to be a feature pitch, for betting using your resources. We may just need to expand on this more to discuss how (or how the following sections) are going to help you secure your infrastructure / applications.